gnome-image-installer/pages/diskimage/gis-diskimage-page.c

```diff
@@ -70,7 +70,8 @@ enum {
     IMAGE_SIZE_BYTES,
     IMAGE_FILE,
     IMAGE_SIGNATURE,
-    ALIGN
+    ALIGN,
+    IMAGE_REQUIRED_SIZE
```

Comment:
Nitpick: I’d add a trailing comma here to reduce diff noise in future. (But there’s no need to fix this now; just for future pondering.)